.vortex/CLAUDE.md

```
@@ -57,6 +57,36 @@ vortex/
   - `_helper.bash` - Test helper functions
   - `fixtures/` - Test fixture files
 
+#### BATS Helpers System
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)